projects/packages/my-jetpack/_inc/components/product-cards-section/videopress-card/videopress-value-section.tsx

```diff
@@ -0,0 +1,22 @@
+import type { FC } from 'react';
+
+import './style.scss';
+
+interface VideoPressValueSectionProps {
+	isPluginActive: boolean;
+	data: Window[ 'myJetpackInitialState' ][ 'videopress' ];
+}
+
+const VideoPressValueSection: FC< VideoPressValueSectionProps > = ( { isPluginActive, data } ) => {
+	if ( ! isPluginActive && data.videoCount ) {
+		return (
+			<div className={ 'videopress-card__video-count' }>
+				<span>{ data.videoCount }</span>
+			</div>
+		);
+	}
+
+	return <></>;
```

Comment:
Why not null?

Reply:
It's just a habit to return some sort of component, shouldn't make any difference but I'll update it to null 😄